spec/spec_helper_acceptance.rb

```diff
@@ -5,6 +5,6 @@
 
 require 'voxpupuli/acceptance/spec_helper_acceptance'
 
-configure_beaker(modules: :metadata)
+configure_beaker
```

Comment:
I don't understand why not write explicitly.

This code depends now on default of voxpupuli-acceptance